tools/python-frame-reference.py

```
@@ -231,57 +254,188 @@ def _build_default_path_vector() -> tuple[dict, str]:
+            "name": "header_overrun",
+            "frame_hex": "434b03000000ff7b7d",
+            "error": "declared header length (255) exceeds the bytes present in the frame",
+        },
+    ]
+    for vec in built_errors:
+        try:
+            wrapper.unwrap(bytes.fromhex(vec["frame_hex"]))
+        except ValueError:
+            pass
+        else:  # pragma: no cover - generation-time invariant
+            raise AssertionError(f"cachekit-py accepted error vector {vec['name']}")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)